ci/cloudbuild/builds/lib/integration.sh

```diff
@@ -271,6 +274,9 @@ function integration::ctest_with_emulators() {
 
   local cmake_out="$1"
   mapfile -t ctest_args < <(ctest::common_args)
+  # The integration tests are inherently flaky.  Try them 3 times before
+  # declaring a failure.
```

Comment:
Nit:  Make the two comments the same so that it is clearer they have the same purpose?

Reply:
Fixed and move the setting to a more general place.